cmd/test.go

```diff
@@ -177,6 +195,17 @@ func init() {
 		},
 	})
 
+
+	Test.AddCommand(&cobra.Command{
+		Use:   "thanos",
+		Short: "Test thanos. Requires Pushgateway and Thanos endpoints",
+		Long: "test thanos [pushgateway] [thanos]",
```

Comment:
I think make this named arguments e.g. `--pushgateway` or `--thanos` and use the local instances if not specified

Reply:
Ok, will do